### PR TITLE
Allow action sheet to stay open after clicking items sometimes

### PR DESCRIPTION
**Pull request by natebunnyfield**

### CODE REVIEW

File: lib/action-sheet/index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ var ActionSheet = React.createClass({
     this.setState({ active: active });
   },
   onBodyClick: function onBodyClick(e) {
-    this.setActiveState(false);
+    if (!e.target.dataset['keepActionSheetActive']) this.setActiveState(false);
```
Comment (antifun) [inline lib/action-sheet/index.js]: Seems like a very not-react way to do this, but then again, it is a hack on top of foundation-apps.

**Comment by natebunnyfield** [comment]
[Create tag front-end](https://trello.com/c/K41pchFT/1610-create-tag-front-end)

File: lib/action-sheet/index.js (hunk 1 above)
Comment (natebunnyfield) [reply lib/action-sheet/index.js]: Leaving this as-is for now. Happy to talk about it  at any time.

Comment (natebunnyfield) [reply lib/action-sheet/index.js]: Well, I could do something like the original code that climbed up the dom tree looking for a certain class. Essentially the same approach, but DRYer.

**Pull request merged**